packages/dashboard/src/components/internalDashboard/index.css

```diff
@@ -51,4 +51,36 @@
   border-radius: 50%;
   width: 40px;
   height: 40px;
+}
+
+.collapsible-panel-vertical-divider {
+  width: 2px;
```

Comment:
can we utilize cloudscape spacing tokens ?

Reply:
I don't believe there are any divider components in cloudscape. Fitting it in to match the mock is difficult with the default padding sizes of SpaceBetween or Box.